frontend/src/pages/MilestonesEditor.tsx

```diff
@@ -257,6 +259,51 @@ export const MilestonesEditor = () => {
       await onVersionDragEnd(result as DropWithDestination);
     }
   };
+
+  const renderTopBar = () => {
+    return (
+      <div className={classes(css.topbar)}>
+        <InfoTooltip
+          title={
+            <div>
+              <Trans i18nKey="Planner milestones tooltip">
+                Milestones create your project’s
+                <Link to={`/roadmap/${roadmapId}/planner/graph`}>roadmap.</Link>
+              </Trans>
+            </div>
+          }
+        >
+          <div className={classes(css.infoBackground)}>
+            <InfoIcon className={classes(css.tooltipIcon, css.infoIcon)} />
+          </div>
+        </InfoTooltip>
+        <div className={classes(css.topbarShares)}>
+          <div>
+            <Trans i18nKey="Actual weighted shares" />
+          </div>
+          {roadmapsVersions && (
+            <TaskValueCreatedVisualization
+              versions={[...roadmapsVersions]}
+              width={250}
+              height={30}
+              barWidth={30}
+            />
+          )}
+        </div>
+        <div className={classes(css.topbarShares)}>
+          <Trans i18nKey="Target weighted shares" />
+          {roadmapsVersions && (
+            <CustomerWeightsVisualization
+              width={250}
+              height={30}
+              barWidth={30}
+            />
+          )}
```

Comment:
The design seems a bit off compared to figma. I'd make the height and width of the bars a little smaller, for example 15px-20px and 200px, as they currently take away the main focus on the page.

```suggestion
          {roadmapsVersions && (
            <TaskValueCreatedVisualization
              versions={[...roadmapsVersions]}
              width={200}
              height={20}
              barWidth={15}
            />
          )}
        </div>
        <div className={classes(css.topbarShares)}>
          <Trans i18nKey="Target weighted shares" />
          {roadmapsVersions && (
            <CustomerWeightsVisualization
              width={200}
              height={20}
              barWidth={15}
            />
          )}
```